ui/admin/app/templates/scopes/scope/targets/target/workers.hbs

```diff
@@ -0,0 +1,131 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.target.workers.title')}}
+<Breadcrumbs::Item
+  @text={{t 'resources.target.workers.title'}}
+  @route='scopes.scope.targets.target.workers'
+/>
+
+<Rose::Layout::Page as |page|>
+
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <Targets::Target::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <Targets::Target::Nav @model={{@model}} />
+  </page.navigation>
+
+  <page.actions>
+    <Targets::Target::Actions
+      @model={{@model}}
+      @delete={{this.targets.delete}}
+    />
+  </page.actions>
+
+  <page.body class='target-workers'>
+    <Hds::Text::Body @tag='p' @size='100' @color='faint'>
+      {{t 'resources.target.workers.description'}}
+    </Hds::Text::Body>
+    <WorkerDiagram
+      @egressWorkerFilterEnabled={{@model.egress_worker_filter}}
+      @ingressWorkerFilterEnabled={{@model.ingress_worker_filter}}
+    />
+    <Hds::Accordion as |A|>
+      {{#if (feature-flag 'target-worker-filters-v2-ingress')}}
+        <A.Item data-test-target-ingress-workers-accordion-item>
+          <:toggle>{{t
+              'resources.target.workers.ingress-workers.title'
+            }}</:toggle>
+          <:content>
+            {{#if @model.ingress_worker_filter}}
+              <Hds::CodeBlock
```

Comment:
Curious on your thoughts on this: I was initially confused because I couldn't find how to edit this field and didn't realize I still had to go through the `Add ingress worker field` dropdown option.

Should we have something here that lets a user go the edit view?

Reply:
thats a very good point. I just checked and accordions do allow for links and buttons inside them. I'll reach out to design to confirm with them.